src/App.tsx

```diff
@@ -30,10 +35,18 @@ const Bar = styled.div`
   }
 `;
 
-interface UserSecurity extends User {
+interface UserExtended extends User {
   readonly signature?: string | undefined;
 }
 
+interface MergeTagsConfig {
+  readonly sort?: boolean | undefined;
+}
+
+interface UnlayerOptionsExtended extends UnlayerOptions {
+  readonly mergeTagsConfig?: MergeTagsConfig | undefined;
+}
+
```

Comment:
What do you think about simply use casting in place of extending the interfaces?

Reply:
Corrected.